docs/topics/composite-primary-key.txt

```diff
@@ -41,8 +41,8 @@ A composite primary key is represented by a ``tuple``:
     >>> item.pk
     (1, "A755H")
 
-You can assign a ``tuple`` to a composite primary key. This sets the associated
-field values.
+You can assign a ``tuple`` to the :attr:`~django.db.models.Model.pk` property.
```

Comment:
Unsure if we should refer to this as a `pk` field or a property, as although I get that you're referencing is as a property here due to the docs link, elsewhere we refer to it as a field

Reply:
I wondered about this also, but throughout this [section](https://docs.djangoproject.com/en/4.2/ref/models/instances/#the-pk-property) it says "property". WDYT?

Reply:
> elsewhere we refer to it as a field

I think the main one in conflict is the instruction to set the CompositePrimaryKey to pk:
>  when creating a model set the ``pk`` field 

I would suggest changing that to property. Then it's more consistent, and helps you grok the rest of the page which explains all the ways `pk` does _not_ act like a field for form and model validation and relations.

(Then there is a reference to the `models.CompositePrimaryKey(Field)` as a virtual field, which it is, but that's not inconsistent with `pk` being a property.)

Reply:
Sounds good, we should also update the release note as it also says "set the pk field"

Reply:
There's one more field reference in the `CompositePrimaryKey` definition we might want to change:
```
This field must be defined as the model's ``pk`` field.
```

I think I want another pair of eyes in case other folks have good ideas (cc @csirmazbendeguz @LilyFoote @charettes )

Reply:
Maybe we should use `attribute` here to disambiguate?